tests/xilinx/adffs.ys

```diff
@@ -0,0 +1,14 @@
+read_verilog adffs.v
+proc
+async2sync # converts async flops to a 'sync' variant clocked by a 'super'-clock
```

Comment:
If this is an attempt to test mapping of asynchronous flipflops, then the `async2sync` here defeats that purpose by making the flipflops synchronous.

In any case, if you did want to test asynchronous flipflops properly, the Xilinx arch doesn't support both async set and reset so these would be left unmapped.

Reply:
Xilinx Vivado synthesizes FDCE cells that have asynchronous reset for this case. And when run synth_xilinx without async2sync it also synthesizes FDCE cells, but I have an: "ERROR: Found 8 unproven $equiv cells in 'equiv_status -assert'" from equiv_opt -assert command. And this test fails.

Reply:
Does Yosys have any optimizations for this case? Vivado synthesizes two ffs and each of them have one control.

Reply:
See #1370 for a solution to the equivalence checking issue by adding `-multiclock` to `equiv_opt`.

> "Xilinx arch doesn't support both async set and reset" Does Yosys have any optimizations for this case? Vivado synthesizes two ffs and each of them have one control.

The solution Vivado uses is a horrible hack involving a LUT used as a latch, iirc, to work around the lack of native support. But I don't think this is something we want to emulate in Yosys, at least by default.

Reply:
OK. Thank you for your description. Will review this test. Maybe will remove both set/reset usage and will add synchronous controls.

Reply:
Sorry `async2sync` was my suggestion. I think what I really wanted was `clk2fflogic` which is dealt with in #1370.